test/hotspot/jtreg/containers/cgroup/CgroupSubsystemFactory.java

```diff
@@ -168,7 +169,7 @@
             "perf_event  4   1   1\n" +
             "net_prio    5   1   1\n" +
             "hugetlb 6   1   1\n" +
-            "pids    3   80  1";
+            "pids    9   80  1";  //  the 3 did not match 9
```

Comment:
This comment leaves the reader none the wiser. I think you are alluding to controller id matching between `/proc/cgroups` and `/proc/self/cgroup`. If so, please use that info.